src/components/RefRole.js

```diff
@@ -18,7 +18,7 @@ const RefRole = ({ nodeData: { children, fileid, target, url }, slug }) => {
     // Render internal target links
     const link = fileid === slug ? `#${target}` : `${fileid}#${target}`;
     return (
-        <Link href={link}>
+        <Link href={link} className="ref-role-anchor">
```

Comment:
Just curious where you pulled this class from? Rendered content?

Reply:
Oh, I meant to call this out. It's helpful to debug the various links if we have readable class names, so I just made this up.